src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/RetryExecutor.kt

```diff
@@ -36,6 +38,7 @@ object RetryExecutor {
         when {
           e::class in exceptions -> {
             lastException = e
+            log.debug("Retrying on error: ${e.message}")
```

Comment:
Won't log in envs as they are set to info. Good for local development to see if the message is consistently retrying on persistence management errors.